tbx/static_src/sass/components/_avatar.scss

```diff
@@ -2,17 +2,19 @@
 
 .avatar {
     position: relative;
-    width: 72px;
-    height: 72px;
+    width: $avatar-size-small;
+    height: $avatar-size-small;
 
     @include media-query(medium) {
-        width: $avatar-size-desktop;
-        height: $avatar-size-desktop;
+        width: $avatar-size-large;
+        height: $avatar-size-large;
```

Comment:
Renamed this and added two others which should be enough to cover all sizes in the design (If not this naming convention will allow us to add more).